### PR TITLE
this pr updates the cloud parity table 

### PR DESCRIPTION
**Pull request by mirnawong1**
discovery api is now supported in azure st and this pr updates the table to reflect that update.

### CODE REVIEW

File: website/snippets/cloud-feature-parity.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The following table outlines which dbt Cloud features are supported on the diffe
 | dbt Cloud IDE                 | ✅           | ✅                     | ✅                  |
 | dbt Cloud CLI                 | ✅           | ✅                     | ❌                  |
 | Audit logs                    | ✅           | ✅                     | ✅                  |  
-| Discovery API                 | ✅           | ✅                     | ❌                  | 
+| Discovery API                 | ✅           | ✅                     | ✅                  | 
 | dbt Explorer                  | ✅           | ✅                     | ❌                  |
```
Comment (mirnawong1) [inline website/snippets/cloud-feature-parity.md]: is dbt explorer also available in azure st? @nghi-ly mentioned that @runleonarun was looking into it: https://dbt-labs.slack.com/archives/C04559AB7D2/p1714489073253449?thread_ts=1714488772.813649&cid=C04559AB7D2

Comment (mirnawong1) [reply website/snippets/cloud-feature-parity.md]: cameron confirmed this is available in ST

Comment (mirnawong1) [inline website/snippets/cloud-feature-parity.md]:
```suggestion
| dbt Explorer                  | ✅           | ✅                     | ✅                   | 
```

File: website/snippets/cloud-feature-parity.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ The following table outlines which dbt Cloud features are supported on the diffe
 | dbt Cloud IDE                 | ✅           | ✅                     | ✅                  |
 | dbt Cloud CLI                 | ✅           | ✅                     | ❌                  |
 | Audit logs                    | ✅           | ✅                     | ✅                  |  
-| Discovery API                 | ✅           | ✅                     | ❌                  | 
-| dbt Explorer                  | ✅           | ✅                     | ❌                  | 
+| Discovery API                 | ✅           | ✅                     | ✅                  | 
+| dbt Explorer                  | ✅           | ✅                     | ✅                   |
```
Comment (mirnawong1) [inline website/snippets/cloud-feature-parity.md:10]: confirmed by cameron

**Review by runleonarun — APPROVED** [review]
Looks good and opened [this PR](https://github.com/dbt-labs/docs.getdbt.com/pull/5415) to add to the prereqs for explorer

**Pull request merged**